usr/src/common/smbclnt/smbfs_ntacl.c

```diff
@@ -42,16 +43,18 @@
 #include <sys/vnode.h>
 #include <sys/vfs.h>
 
-#include <sys/kidmap.h>
-
 #else	/* _KERNEL */
 
 #include <stdio.h>
 #include <stdlib.h>
 #include <strings.h>
 
-#include <idmap.h>
+#endif	/* _KERNEL */
```

Comment:
Yeah, probably should update to match the ifdef, here and below.
(Thanks @sjorge)